src/main/utils.js

```diff
@@ -22,17 +23,17 @@ export function shouldBeHiddenOnStartup(parsedArgv) {
   return false;
 }
 
-export function getWindowBoundaries(win) {
+export function getWindowBoundaries(win, hasBackBar = false) {
   const {width, height} = win.getContentBounds();
-  return getAdjustedWindowBoundaries(width, height);
+  return getAdjustedWindowBoundaries(width, height, hasBackBar);
 }
 
-export function getAdjustedWindowBoundaries(width, height) {
+export function getAdjustedWindowBoundaries(width, height, hasBackBar = false) {
   return {
     x: 0,
-    y: TAB_BAR_HEIGHT,
+    y: TAB_BAR_HEIGHT + (hasBackBar ? BACK_BAR_HEIGHT : 0),
     width,
-    height: height - TAB_BAR_HEIGHT,
+    height: height - TAB_BAR_HEIGHT - (hasBackBar ? BACK_BAR_HEIGHT : 0),
```

Comment:
I know this was the case before, but why are we removing the bars height?

Reply:
We want the browser view to be the same size as the window, but without the height of the topbar (or in this case, the back bar as well) included.

eg. Window is 500px tall, tab bar is 40px and the backbar is 30px. So we'd want the browser to be rendered at the bottom of the backbar, at a height of 430px.